docs/userguide/datafiles.rst

```diff
@@ -87,14 +88,13 @@ When building an ``sdist``, the datafiles are also drawn from the
 ``package_name.egg-info/SOURCES.txt`` file, so make sure that this is removed if
 the ``setup.py`` ``package_data`` list is updated before calling ``setup.py``.
 
-(Note: although the ``package_data`` argument was previously only available in
-``setuptools``, it was also added to the Python ``distutils`` package as of
-Python 2.4; there is `some documentation for the feature`__ available on the
-python.org website.  If using the setuptools-specific ``include_package_data``
-argument, files specified by ``package_data`` will *not* be automatically
-added to the manifest unless they are listed in the MANIFEST.in file.)
+.. note::
+   If using the ``include_package_data`` argument, files specified by
+   ``package_data`` will *not* be automatically added to the manifest unless
+   they are listed in the |MANIFEST.in|_ file or by a plugin like
+   :pypi:`setuptools-scm` or :pypi:`setuptools-svn`.
```

Comment:
I don't know if this affirmation still holds, I have the impression that sometimes ``package_data`` is enough and does not require ``MANIFEST.in``...

This does seem to be the case in the experiment performed in: https://github.com/abravalheri/experiment-setuptools-integration-tests

Reply:
THANK YOU for that repo with the experiments, it is super helpful!